_includes/toc.html

```diff
@@ -0,0 +1,218 @@
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
+    WARNING - THIS FILE HAS BEEN MODIFIED MANUALLY BY VRCHAT API - DO NOT UPGRADE WITHOUT DUE DILIGENCE
```

Comment:
Unnecessary repeat?

Reply:
Repeat has now been reduced to 5 lines instead of the entire page. This should be sufficient to prevent someone from accidentally modifying or updating it without accidental feature regression.